doc/classes/EditorSettings.xml

```diff
@@ -565,6 +565,9 @@
 			If [code]true[/code], editor main menu is using embedded [MenuBar] instead of system global menu.
 			Specific to the macOS platform.
 		</member>
+		<member name="interface/inspector/auto_enable_vector_property_linking" type="bool" setter="" getter="">
```

Comment:
I would name the setting `enable_vector_property_linking_by_default`, so that it looks more natural when capitalized in the Editor Settings.

Reply:
Only temporary, as seen in the bug